generators/app/templates/ext-command-ts/vscodeignore

```diff
@@ -6,3 +6,4 @@ src/**
 .gitignore
 tsconfig.json
 vsc-extension-quickstart.md
+tslint.json
```

Comment:
Can also add 'extensions.json'

Reply:
It is already excluded by the line `.vscode/**` at the top of the file.